addon/scope.js

```diff
@@ -0,0 +1,60 @@
+class Scope {
```

Comment:
Is the test registry available?

If so, that would cleanup the current implementation as well.

The only downside is that there are some helpers that **don't** accept a selector (i.e. `visit`, `currentURL`, etc).

We could just do it anyway and hope that our users would never call those methods.

What do you think?